.gitignore

```diff
@@ -105,3 +105,6 @@ venv.bak/
 
 # Hide ckanapi installation
 src/
+
+# Hide translation .po file copies
+translations/**/
```

Comment:
maybe make it a subdirectory like `build` to indicate those are artefacts outputs safe to remove.
e.g.
```
build/translations/*
```